addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.scss

```diff
@@ -59,6 +59,10 @@
     border-top-color: transparent;
 }
 
+.o_ChatterTopbar_button:hover {
+    background-color: lighten(gray('300'), 7%);
```

Comment:
Isn't is possible to pick a lighter gray directly, instead of applying transformation?

Reply:
Don't know, just took the same gray as active button

Reply:
After testing (in community, but it shouldn't matter) I feel like it is too subtle, and it is the same background color as the "selected tab" (message/note) so I would suggest something different/darker for hover.